src/task/cleanup.php

```diff
@@ -26,28 +25,32 @@
             continue;
```

Comment:
instead of retrieving the current release from its path, wouldn't it make more sense to skip `0` as $releasesList[1] is the previous release in deployer, so 0 should be the current.

or compare the release path to the target release path instead of checking the number.